src/uct/ib/base/ib_md.c

```diff
@@ -147,6 +148,14 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
      "For example a filter for the default subnet prefix can be specified as: fe80:0:0:0",
      ucs_offsetof(uct_ib_md_config_t, subnet_prefix), UCS_CONFIG_TYPE_STRING},
 
+    {"SUBNET_MASK", "0",
```

Comment:
Why do we need a configuration for this? The system net config already defines the subnet prefix of the RoCE interface

Reply:
Because what we have today as subnet prefix does not match my requirements:
````
    {"SUBNET_PREFIX", "",
     "Infiniband subnet prefix to filter ports by, empty means no filter. "
     "Relevant for IB link layer only\n"
     "For example a filter for the default subnet prefix can be specified as: fe80:0:0:0",
     ucs_offsetof(uct_ib_md_config_t, subnet_prefix), UCS_CONFIG_TYPE_STRING},
````
1. it is IB-only (I need RoCE)
2. it's a string (I need number of bits)
3. used to filter for an explicit prefix specified by the user (I need to compare local and remote IPs)

Reply:
agreed, it makes more sense to ask linux for the netmask...

Reply:
@yosefe - looks like Verbs API doesn't give netmask information, and the only way I see is using SIOCGIFNETMASK like TCP transport does... would that be acceptable for RoCE devices in IB transport?

Reply:
yes, need to extract from system (add it to ucs/sys/sock.c), potentially need to use other method so it will work on ipv6 as well